Code/GraphMol/Descriptors/DCLV.cpp

```diff
@@ -295,7 +295,7 @@ struct State {
       if (!(vert = equat >> 1)) vert = 1;
 
       unsigned int i = 0;
-      for (unsigned int j = 0; (i < count) && (j < vert); j++) {
+      for (unsigned int j = 0; std::cmp_less(i, count) && (j < vert); j++) {
```

Comment:
I think it would make a lot more sense to just change the type of `count` to `unsigned long`